traits/metricstrait/config/samples/application/README.md

```diff
@@ -0,0 +1,61 @@
+# Deployment Workload
```

Comment:
we should move this to  `traits/metricstrait/README.md`, or user can't view it in this place.

Reply:
it was designed for this example only and it is pretty much a "quick start". Maybe I can add a main  README that points to this? I can put more context in the main README.

Reply:
Yes, add a  main  README that points to this is good!